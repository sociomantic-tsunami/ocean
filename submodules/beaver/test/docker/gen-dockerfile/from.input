FROM user/repo:tag
FROM user/repo:tag # comment FROM baduser/badrepo:badtag
FROM user/repo:tag# comment FROM baduser/badrepo:badtag
FrOm user/repo:tag
FROM user/repo
FROM repo
FROM repo:tag
     	FROM   user/repo:tag    	#   comment
FROM user/repo:v3.0.1-rc_1
FROM user/repo#:tag
FROM user#/repo:tag
FROM user # /repo:tag
