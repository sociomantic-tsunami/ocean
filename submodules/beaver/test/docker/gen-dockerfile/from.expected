FROM user/repo:xenial-tag
FROM user/repo:xenial-tag # comment FROM baduser/badrepo:badtag
FROM user/repo:xenial-tag# comment FROM baduser/badrepo:badtag
FrOm user/repo:xenial-tag
FROM user/repo
FROM repo
FROM repo:xenial-tag
     	FROM   user/repo:xenial-tag    	#   comment
FROM user/repo:xenial-v3.0.1-rc_1
FROM user/repo#:tag
FROM user#/repo:tag
FROM user # /repo:tag
